services/manage/app/templates/manage/orders/edit.hbs

```diff
@@ -26,6 +26,11 @@
           {{lineitem/-icon}} Line Items <span class="badge badge-light">{{model.lineitems.totalCount}}</span>
         {{/link-to}}
       </li>
+      <li class="nav-item">
+        {{#link-to "manage.orders.edit.links"  class="nav-link" role="tab"}}
+          {{entypo-icon "link"}} Links
```

Comment:
Turn this into an `external-link/-icon` component to be consistent with the other icon comps